lib/src/header/Header.tsx

```diff
@@ -303,34 +311,35 @@ const ResponsiveMenu = styled.div`
 const ResponsiveLogoContainer = styled.div`
   max-height: ${(props) => props.theme.logoHeight};
   width: ${(props) => props.theme.logoWidth};
-  position: absolute;
-  top: 23px;
-  left: 20px;
+  display: flex;
+`;
+
+const ResponsiveIconsContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
 `;
 
 const CloseContainer = styled.div`
```

Comment:
In the previous review, we had agreed to change the `div` for a `button`.

![image](https://user-images.githubusercontent.com/44321109/189340715-770d2594-cb2a-4de3-80db-7cd11cf18b07.png)

Also, I'm not completely sure on how we are currently styling this with that absolute positioning. How I would do this:

```
const CloseAction = styled.button`
  display: flex;
  flex-wrap: wrap;
  align-content: center;
  padding: 6px;
  height: 36px;
  width: 36px;
  background-color: transparent;
  border: 1px solid transparent;
  border-radius: 2px;

  :focus,
  :focus-visible {
    outline: ${(props) => props.theme.hamburguerFocusColor} auto 1px;
  }
`;
```

Reply:
I would not use a transparent border for that button because it doesn't have any, only the focus outline.
I can see it better like this:
```
const CloseAction = styled.button`
  cursor: pointer;
  width: 24px;
  height: 24px;
  border: none;
  background-color: transparent;
  padding: 0px;
  :focus,
  :focus-visible {
    outline: ${(props) => props.theme.hamburguerFocusColor} auto 1px;
  }
  svg {
    width: 24px;
    height: 24px;
  }
`;
```

Reply:
My thoughts about this are in the review!